internal/util/metrics/metrics.go

```diff
@@ -35,6 +39,10 @@ type CustomMetrics struct {
 	MonoVerticesSyncFailed *prometheus.CounterVec
 	// MonoVerticesSynced is the counter for the total number of monovertices synced.
 	MonoVerticesSynced *prometheus.CounterVec
+	// NumaflowControllersHealth is the gauge for the health of Numaflow controller.
+	NumaflowControllersHealth *prometheus.GaugeVec
+	// NumaflowControllerVersionCounter contains the information of all running numaflow controllers with "version" as a key and "name_namespace" as a value.
```

Comment:
there are a few comments which say "name_namespace" - should they?

Reply:
Ohh, It should not be. Might left the comment by mistake.
Fixed, Thanks